src/Model.php

```diff
@@ -2,9 +2,9 @@
 
 namespace Musonza\Chat;
 
-use Eloquent;
+use Illuminate\Database\Eloquent\Model;
 
-class Model extends Eloquent
+class Model extends Model
```

Comment:
This won't work because now you have the base `Model` and also importing `Model` which is not aliased. You can rename ` Musonza\Chat\Model` to ` Musonza\Chat\BaseModel` and have every model extend BaseModel. That way you won't need to add `use Illuminate\Database\Eloquent\Model` in every model

Reply:
Yea right, I will fix that tonight and commit it. 
Greets axe.